scripts/ci/build_emscripten.sh

```diff
@@ -40,10 +40,7 @@ else
 	BUILD_DIR="$1"
 fi
 
-apt-get update
-apt-get install lz4 --no-install-recommends
-
-WORKSPACE=/root/project
+WORKSPACE=/emsdk/project
```

Comment:
Tests are failing because of this. You probably just need to create the dir. Either inside or outside the image. Or is docker supposed to create it automatically?

Reply:
I thought that docker run would create the volume if it does not exists in the [container](https://docs.docker.com/storage/volumes/#start-a-container-with-a-volume). Maybe this script is being called directly by another script I did not see. I will check.